reflex/config.py

```diff
@@ -726,13 +727,34 @@ def __init__(self, *args, **kwargs):
                 "REDIS_URL is required when using the redis state manager."
             )
 
+    @property
+    def app_module(self) -> ModuleType | None:
+        """Get the app module.
+
+        Returns:
+            The app module.
+        """
+
+        def load_via_spec(path):
+            module_name = Path(path).stem
+            spec = importlib.util.spec_from_file_location(module_name, path)
+            module = importlib.util.module_from_spec(spec)
+            spec.loader.exec_module(module)
+            return module
+
+        if self.app_module_path:
+            return load_via_spec(self.app_module_path)
+        return None
+
     @property
     def module(self) -> str:
         """Get the module name of the app.
 
         Returns:
             The module name.
         """
+        if self.app_module:
+            return f"{Path(self.app_module.__file__).parent.name}.{Path(self.app_module.__file__).stem}"
```

Comment:
wondering if this is what we should be doing. Or should we return the absolute path here? Im trying to think of where doing it like this could pose a real problem